README.md

```diff
@@ -1,4 +1,2 @@
 # project-thor
-This is a project to creating an entire web page based on a design provided below;
-
-![Alt text](image-01.png)
```

Comment:
The image below should always have the summary of what is done in the PR:
![image](https://user-images.githubusercontent.com/10332499/205148832-ea83c025-7b38-4367-be58-6f4f6395467b.png)